src/services/location.js

```diff
@@ -128,11 +128,12 @@ ngeo.Location.prototype.getParam = function(key) {
 /**
  * Get a param from the fragment of the location's URI.
  * @param {string} key Param key.
- * @return {string} Param value.
+ * @return {string|undefined} Param value.
  * @export
  */
 ngeo.Location.prototype.getFragmentParam = function(key) {
-  return /** @type {string} */ (this.getFragmentUri_().getQueryData().get(key));
+  var val = /** @type {string} */ (this.getFragmentUri_().getQueryData().get(key));
+  return val !== undefined ? goog.string.urlDecode(val) : undefined;
```

Comment:
Can we get ride of the goog library?
See https://github.com/google/closure-library/blob/master/closure/goog/string/string.js#L527